.vscode/launch.json

```diff
@@ -26,6 +26,9 @@
             "preLaunchTask": "", //"build",
             "program": "${workspaceFolder}/out/dotnet/source/Mlos.Agent.Server/objd/AnyCPU/Mlos.Agent.Server.dll",
             "args": [
+                //"--optimizer-uri",
+                //"http://localhost:50051",

```

Comment:
Fixup for the default debugger experience.
If this options are enabled by default, the command will fail.

In the future we can consider to add a `preLaunchTask` that starts the optimizer service.